openedx/frontend-app-course-authoring
# fix: transcript tab layout

JIRA Ticket: [TNL-11200](https://2u-internal.atlassian.net/browse/TNL-11200)

> The list of transcript stretches indefinitely. Update to make the list of languages scrollable and alphabetical order. To help with reducing the space, update the size of the dropdown text.
> Update modal to have the following layout
> ![image](https://github.com/openedx/frontend-app-course-authoring/assets/42981026/64033528-4dfc-4a8f-9c71-3edd1a16c561)

Testing

1. Navigate to the Files page
2. Open the info modal for any file
3. Should render with no change to the ui
4. Navigate to the Videos page
5. Open the info modal for any video with transcripts
6. Click on the transcript tab
7. Transcripts should load in alphabetical order
8. Add a new transcript
9. Should add empty transcript to the top and scroll it into view
10. Add multiple new transcripts so that the tab is forced to scroll
11. Upload a transcript to one of the empty rows
12. New transcript should move into the correct alphabetical location.
13. Delete extra empty rows.
